ports/servo/main.rs

```diff
@@ -143,7 +145,10 @@ fn main() {
     // Our wrapper around `Browser` that also implements some
     // callbacks required by the glutin window implementation.
     let mut browser = BrowserWrapper {
-        browser: Browser::new(window.clone()),
+        browser: Browser::new(window.clone(),
+                opts::get().url.clone()
+                    .unwrap_or(ServoUrl::parse(PREFS.get("shell.homepage").as_string()
+                        .unwrap_or("about:blank")).unwrap()))
```

Comment:
Can you split that in a way that it's easier to understand?
Maybe move the url resolution above, and add a comment to explain the logic.